Framework/PythonInterface/plugins/algorithms/WorkflowAlgorithms/FlatPlatePaalmanPingsCorrection.py

```diff
@@ -315,10 +318,10 @@ def _add_sample_logs(self, ws, sample_logs):
         @param sample_logs Dictionary of logs to append to the workspace.
         """
 
-        for key, value in sample_logs.iteritems():
+        for key, value in iteritems(sample_logs):
             if isinstance(value, bool):
                 log_type = 'String'
-            elif isinstance(value, (int, long, float)):
+            elif isinstance(value, (int, float)):
```

Comment:
It could still be `long` in python 2. So you should `from six import integer_types` then here have `isinstance(value, (integer_types, float))`